### ISSUE TITLE
The current page in the Pager buttons should have "aria-current='page'" or we need to replace the anchor tags with span elements

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-react#1324]
For accessibility, the selected page number should have "aria-current='page'" attribute to the anchor element or we should replace the anchor tags with span (as in the Kendo UI for jQuery: [https://demos.telerik.com/kendo-ui/grid/keyboard-navigation](https://demos.telerik.com/kendo-ui/grid/keyboard-navigation))

Currently, we are only adding "k-selected" class to the currently selected page.

**Comment by kdikov82** [comment]
Resolved in latest versions (tested with 5.11.0)
